src/robusta/core/sinks/robusta/robusta_sink.py

```diff
@@ -116,6 +118,25 @@ def __publish_new_services(self, active_services: List):
         # save the cached services in the resolver.
         TopServiceResolver.store_cached_services(list(self.__services_cache.values()))
 
+    def __get_events_history(self):
+        try:
+            # we will need the services in cache before the event history is run to guess service name
+            self.__discover_services()
+            response = WebApi.run_manual_action(
+                action_name="event_history",
+                params=None,
+                sinks=[self.sink_name],
+                retries=4,
+                timeout_delay=5
```

Comment:
I think total of 20 seconds might not be enough.
I would choose something bigger